src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/FixedPolicy.cs

```diff
@@ -2,30 +2,47 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Linq;
 
 namespace Azure.Base.Http.Pipeline
 {
-    class FixedPolicy : RetryPolicy {
-        int _maxRetries;
-        TimeSpan _delay;
-        int[] _retriableCodes;
+    internal class FixedPolicy : RetryPolicy
```

Comment:
nit:  I'd recommend renaming to `FixedRetryPolicy` to make its intended purpose clear at a glance without needing to open the file and look at the inheritance.